modules/users/client/views/profile/profile-view.client.view.html

```diff
@@ -134,7 +134,7 @@ <h1>Don't panic!</h1>
 
     <div class="row">
 
-      <div class="col-sm-3 hidden-xs" ng-include="'/modules/users/views/profile/profile-view-basics.client.view.html'"></div>
+      <div class="col-sm-3" ng-include="'/modules/users/views/profile/profile-view-basics.client.view.html'"></div>
```

Comment:
Hello @PaulElis, nice to have you here! :tada: 

With the proposed change we show more than we wanted. Now I can see avatar twice on small screens in profile `Overview`.
I'd imagine a way forward could rather be adding a margin at the bottom for the small screens (to give the `report member some space at the bottom`) or something similar.